README.md

```diff
@@ -82,6 +82,41 @@ The Hamler 0.1 compiler was forked from [PureScript][PureScriptHamler] 0.13.6. T
   make install
   ```
 
+**Build from source code(linux/macOS)**
+
+0. Install [asdf-vm](https://asdf-vm.com/#/core-manage-asdf-vm?id=install)
+
+1. Install asdf-vm [Erlang plugin](https://github.com/asdf-vm/asdf-erlang)
+
+  ```shell
+  asdf plugin-add erlang
+  ```
+
+3. Clone hamler from the git repo and change directory
```

Comment:
I think the sequence number of the ordered list should be correct. Although the sequence number rendered by markdown is correct, we should also keep the text content correct

Reply:
Right. I've addressed this as well.

Reply:
Thx :)